bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-data-model-fragments",
   "devDependencies": {
-    "ember": "~2.4.3",
+    "ember": "~2.5.0",
```

Comment:
@slindberg Yeap, I think that's a good idea. I'll submit a PR for this right now.